turbinia/client.py

```diff
@@ -95,6 +95,24 @@
 logger.setup()
 
 
+def get_turbinia_client(run_local=False):
+  """Return Turbinia client based on config.
+
+  Returns:
+    Initialized BaseTurbiniaClient or TurbiniaCeleryClient object.
+  """
+  config.LoadConfig()
+  # pylint: disable=no-else-return
+  if config.STATE_MANAGER.lower() == 'datastore':
```

Comment:
Small nit, but could you potentially make this check for `celery` or `psq` from the `TASK_MANAGER` variable instead of the `STATE_MANAGER`?  From a practical standpoint it probably doesn't matter much right now because when we use redis we also use celery, but I'd like to start allowing the cloud version use redis sometime soon, and so I'm trying to keep the  state manager and the task manager separate in the code base so we don't miss some code locations later.  Thanks!